store/tikv/coprocessor.go

```diff
@@ -920,6 +920,12 @@ func (worker *copIteratorWorker) handleCopResponse(bo *Backoffer, rpcCtx *RPCCon
 		resp.detail = new(execdetails.ExecDetails)
 	}
 	resp.detail.BackoffTime = time.Duration(bo.totalSleep) * time.Millisecond
+	resp.detail.BackoffSleep, resp.detail.BackoffTimes = make(map[string]time.Duration), make(map[string]int)
+	for backoff := range bo.backoffTimes {
+		backoffName := backoff.String()
+		resp.detail.BackoffTimes[backoffName] = bo.backoffTimes[backoff]
+		resp.detail.BackoffSleep[backoffName] = time.Duration(bo.backoffSleepMS[backoff]) * time.Millisecond
+	}
```

Comment:
do we need take care unsuccessful response, for example back off some times and return at 885

Reply:
It seems we don't need to take care of it since if it returns at line:885, the caller will ignore the `resp` and just return the error to the outside at line:720.

Reply:
Updated, it is `Cop_backoff_` now.